applications/plugins/OpenCTMPlugin/src/SofaOpenCTM/config.h.in

```diff
@@ -19,15 +19,15 @@
 *                                                                             *
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
-#ifndef OPENCTMPLUGIN_CONFIG_H
-#define OPENCTMPLUGIN_CONFIG_H
+#ifndef SOFA_OPENCTM_CONFIG_H
+#define SOFA_OPENCTM_CONFIG_H
 
 #include <sofa/helper/system/config.h>
 
 #ifdef SOFA_BUILD_OPENCTMPLUGIN
-#  define SOFA_OPENCTMPLUGIN_API SOFA_EXPORT_DYNAMIC_LIBRARY
+#  define SOFA_OPENCTM_API SOFA_EXPORT_DYNAMIC_LIBRARY
 #else
-#  define SOFA_OPENCTMPLUGIN_API SOFA_IMPORT_DYNAMIC_LIBRARY
+#  define SOFA_OPENCTM_API SOFA_IMPORT_DYNAMIC_LIBRARY
 #endif
```

Comment:
`SOFA_OPENCTMPLUGIN_API` was consistent and respected the SOFA_MODULENAME_API format. I would keep it.

Reply:
In fact I want to change the name of the plugin to SofaOpenCTM (as it is a lib, like SofaCGAL or SofaCUDA) when it will be moved out. 
I already moved the code internally into src/SofaOpenCTM